devicetypes/smartthings/zwave-fan-controller.src/zwave-fan-controller.groovy

```diff
@@ -110,7 +110,9 @@ def fanEvents(physicalgraph.zwave.Command cmd) {
 	def value = (cmd.value ? "on" : "off")
 	def result = [createEvent(name: "switch", value: value)]
 	result << createEvent(name: "level", value: cmd.value == 99 ? 100 : cmd.value)
-	result << createEvent(name: "fanSpeed", value: Math.round(cmd.value/33))
+	def fan_level = Math.round(cmd.value/33)
+	if (cmd.value < 33 * cmd.value > 1) fan_level = 1 //sometimes we get "1" when the device is on low
```

Comment:
Isn't `cmd.value < 33 * cmd.value > 1` always going to evaluate to `true` unless `cmd.value` is `0`? Would `0 < cmd.value < 33` make more sense?

Reply:
Or I assume this was meant to be `cmd.value < 33 && cmd.value > 0`.

Reply:
you are correct